### PR TITLE
[Amazon Q chat]: Adding TextUtils to core folder for AB testing.

### PR DESCRIPTION
**Pull request by laileni-aws**
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Description
- Existing plugin is using TestUtils from toolkit folder so Standalone extension not able to access this folder.
- Copied this file to core folder.
- Fixed the AB testing number of code blocks param.

## Checklist
- [x] My code follows the code style of this project
- [ ] I have added tests to cover my changes
- [ ] A short description of the change has been added to the **[CHANGELOG](https://github.com/aws/aws-toolkit-jetbrains/blob/master/CONTRIBUTING.md#contributing-via-pull-requests)** if the change is customer-facing in the IDE.
- [x] I have added metrics for my changes (if required)
 
## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by ptrucks — APPROVED** [review]

File: plugins/toolkit/core/src/software/aws/toolkits/core/utils/TestUtils.kt
Comment (rli) [inline plugins/toolkit/core/src/software/aws/toolkits/core/utils/TestUtils.kt:1]: **TEXT**Utils?

File: plugins/toolkit/core/src/software/aws/toolkits/core/utils/TestUtils.kt [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+            html.tag("code", mapOf("class" to "language-$language"))
+            html.text(codeBlock.literal)
+            html.tag("/code")
+            html.tag("/pre")
+        }
+
+        html.tag("/div")
+        html.line()
+    }
+}
```
Comment (rli) [inline plugins/toolkit/core/src/software/aws/toolkits/core/utils/TestUtils.kt:53]: Please stop duplicating code across the code base. If you are moving code into core so it is accessible, you need to delete the original

File: plugins/toolkit/core/build.gradle.kts [hunk 2]
```diff
@@ -23,7 +23,7 @@ dependencies {
 //    api(libs.aws.ssooidc)
     api(libs.aws.sts)
     api(libs.bundles.jackson)
-
+    implementation(libs.commonmark)
```
Comment (rli) [inline plugins/toolkit/core/build.gradle.kts:26]: There is no other usage of commonmark so we can remove the dependency from `toolkit/jetbrains-core`

**Review by rli — CHANGES_REQUESTED** [review]

File: plugins/toolkit/core/src/software/aws/toolkits/core/utils/TestUtils.kt (hunk 1 above)
Comment (laileni-aws) [reply plugins/toolkit/core/src/software/aws/toolkits/core/utils/TestUtils.kt:53]: Original has different other functions which is used in different places. So I thought to better take this into core

File: plugins/toolkit/core/src/software/aws/toolkits/core/utils/TestUtils.kt
Comment (laileni-aws) [reply plugins/toolkit/core/src/software/aws/toolkits/core/utils/TestUtils.kt:1]: Updated!

File: plugins/toolkit/core/build.gradle.kts (hunk 2 above)
Comment (laileni-aws) [reply plugins/toolkit/core/build.gradle.kts:26]: I will move `commonmark` and its related usecase from `toolkit/jetbrains-core` to `core `.

**Review by ashishrp-aws — APPROVED** [review]

**Review by rli — APPROVED** [review]

**Pull request merged**